packages/design-tokens/src/tokens/functional/components/bento/base.json

```
@@ -0,0 +1,43 @@
+{
+  "brand": {
+    "Bento": {
+      "gap": {
+        "large": {
+          "value": "var(--base-size-32)"
+        },
+        "medium": {
+          "value": "var(--base-size-24)"
+        },
+        "small": {
+          "value": "var(--base-size-20)"
+        }
+      },
+      "item": {
+        "borderRadius": {
+          "small": {
+            "value": "var(--base-size-16)"
+          },
+          "medium": {
+            "value": "var(--base-size-20)"
+          },
```

Same comment as above, medium should be 24px

```suggestion
          "medium": {
            "value": "var(--base-size-24)"
          },
```